Parsers/ASimDns/ProductParsers/vimDnsInfobloxNIOS.yaml

```diff
@@ -57,7 +57,7 @@ ParserQuery: |
           (dnsresponsename=='*')
           and (array_length(response_has_any_prefix) ==0) 
           and (isnull(starttime) or TimeGenerated >= starttime)
-          and (isnull(endtime) or TimeGenerated <= endtime) 
+          and (isnull(endtime) or TimeGenerated <= centime) 
```

Comment:
?

Reply:
No clue. Thanks for catching. I did not intend to change it, and therefore did not test the KQL.